datahub-web-react/src/app/preview/EntityPaths/DisplayedColumns.tsx

```diff
@@ -25,7 +26,7 @@ export default function DisplayedColumns({ displayedColumns }: Props) {
                     return (
                         <ColumnNameWrapper>
                             {entity.type === EntityType.SchemaField
-                                ? downgradeV2FieldPath((entity as SchemaFieldEntity).fieldPath)
+                                ? decodeSchemaField(downgradeV2FieldPath((entity as SchemaFieldEntity).fieldPath) || '')
```

Comment:
Should we decode _after_ downgrading or before?

Reply:
I don't think it would really matter as decoding it just converts a few encoded characters back to normal (`(`, `)`, and `,`) and none of those should be affected by the V2 field path. I believe decoding after downgrading would be safest so then we're decoding what we want to show in the UI.